homeassistant/components/zha/core/device.py

```diff
@@ -276,7 +276,7 @@ def is_groupable(self) -> bool:
     @property
     def skip_configuration(self) -> bool:
         """Return true if the device should not issue configuration related commands."""
-        return self._zigpy_device.skip_configuration or self.is_coordinator
+        return self._zigpy_device.skip_configuration or bool(self.is_coordinator)
```

Comment:
`self.is_coordinator` can be `None`, so we need to convert falsey values to a proper boolean.